<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="wjmelly" failures="2" tests="3" name="Submit Order Test" time="37.042" errors="0" timestamp="2025-06-04T16:16:02 WIB">
  <testcase classname="rahulshettyacademy.test.SubmitOrderTest" name="submitOrder" time="12.025">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at rahulshettyacademy.test.SubmitOrderTest.submitOrder(SubmitOrderTest.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- submitOrder -->
  <testcase classname="rahulshettyacademy.test.SubmitOrderTest" name="submitOrder" time="11.765">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at rahulshettyacademy.test.SubmitOrderTest.submitOrder(SubmitOrderTest.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- submitOrder -->
  <testcase classname="rahulshettyacademy.test.SubmitOrderTest" name="OrderHistoryTest" time="0.0">
    <skipped/>
  </testcase> <!-- OrderHistoryTest -->
</testsuite> <!-- Submit Order Test -->
